templates/common/infra/bicep/core/cdn/cdn-endpoint.bicep

```diff
@@ -0,0 +1,49 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+@description('The origin URL for the endpoint')
+@minLength(1)
+param originUrl string
+
+@description('The name of the CDN profile resource')
+@minLength(1)
+param cdnProfileName string
```

Comment:
Is this something we can default?

Reply:
Hm, we could default it to empty string, but then the existing resource definition wouldn't work out at the end. I looked at other examples of modules which parent: a resource from another module, and the names don't have default values. Maybe I'm mis-thinking this though, let me know.

Reply:
Ok